demo/demo-springmvc/springmvc-server/src/main/java/org/apache/servicecomb/demo/springmvc/server/ProducerTestsAfterBootup.java

```diff
@@ -90,11 +89,7 @@ public void testSchemaNotChange(SCBEngine scbEngine) {
   }
 
   public void testRegisteredBasePath() {
-    if (DynamicPropertyFactory.getInstance().getBooleanProperty("servicecomb.test.vert.transport", true).get()) {
-      TestMgr.check(17, RegistrationManager.INSTANCE.getMicroservice().getPaths().size());
-    } else {
-      TestMgr.check(18, RegistrationManager.INSTANCE.getMicroservice().getPaths().size());
-    }
+    TestMgr.check(18, RegistrationManager.INSTANCE.getMicroservice().getPaths().size());
```

Comment:
Not correct, change 17 -> 18, 18 -> 19.  Because this project is shared in demo-spring-boot-transport

Reply:
no wonder it fails in demo-spring-boot-transport test cases -_-''